.github/workflows/traclusdl.yml

```
@@ -0,0 +1,75 @@
+# Workflow to verify the code style of the commit, that it builds correctly
+# and make sure unit tests run too with all supported rust versions
+
+name: CI
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)